src/os/flatpak.zig

```diff
@@ -71,18 +72,28 @@ pub const FlatpakHostCommand = struct {
 
         /// Process started with the given pid on the host.
         started: struct {
-            pid: c_int,
+            pid: u32,
```

Comment:
Is the PID necessarily a u32? pid_t is always c_int AFAICT

Reply:
Yea but Flatpak's `HostCommand` interface outputs `u32`, so I made sure that's followed here.